.github/workflows/prover-daily-test.yaml

```diff
@@ -23,10 +23,10 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
-      - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
+      - uses: aptos-labs/aptos-core/.github/actions/rust-setup-prover-tests@main
       - name: install Prover dependencies
         shell: bash
-        run: scripts/dev_setup.sh -b -p -y -J
+        run: scripts/dev_setup.sh -b -p -y
```

Comment:
can you include this step in the new action?

Reply:
done